cypress/e2e/audit-overview-page.cy.ts

```diff
@@ -0,0 +1,16 @@
+import { testA11y } from 'cypress/support/utils';
+
+describe('Audit Overview Page', () => {
+  beforeEach(() => {
+    // Must login as an Admin to see the page
+    cy.visit('/auditlogs');
+    cy.loginViaForm(Cypress.env('DSPACE_TEST_ADMIN_USER'), Cypress.env('DSPACE_TEST_ADMIN_PASSWORD'));
+  });
+
+  it('should pass accessibility tests', () => {
+    // Page must first be visible
+    cy.get('ds-audit-overview').should('be.visible');
+    // Analyze <ds-audit-overview> for accessibility issues
+    testA11y('ds-audit-overview');
+  });
```

Comment:
I would appreciate it if we could add some more basic e2e tests for this page... even just check that the basic page structure exists.